release.yaml

```diff
@@ -115,3 +115,5 @@ rancher-gatekeeper-crd:
   - 102.1.1+up3.13.0
 neuvector-monitor:
   - 102.0.8+up2.7.3
+rancher-turtles:
+- 100.1.0+up0.5.0
```

Comment:
Please see the following rules for versioning: https://github.com/rancher/charts/blob/dev-v2.8/README.md#versioning-charts

You are trying to merge at dev-v2.8 branch.